### PR TITLE
Print full board

### PR DESCRIPTION
**Pull request by jerivasuaq**

### CODE REVIEW

**Review by jerivasuaq — COMMENTED** [review]
Nice coding

**Review by JuanPer98 — APPROVED** [review]

**Review by Daniel-Rocha-R — APPROVED** [review]

**Review by Frank-Garer — APPROVED** [review]
Good job! But I think it needs some raptors!!

**Review by DarioGald — APPROVED** [review]

**Review by Kri3gtark — CHANGES_REQUESTED** [review]
Add Pices for Player 2

**Review by vmontalvo-coder — APPROVED** [review]

**Review by Frank-Garer — CHANGES_REQUESTED** [review]
Where are the second player's pieces?

**Review by MSmartine01 — CHANGES_REQUESTED** [review]
player 2 is missing

File: chessboard.py [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
         self.board = [
-            [Rook(0,0), Knight(0,1), Bishop(0,2)],
-            [Pawn(0,0), Pawn(0,1), Pawn(0,2)],
+            [Rook(0,0), Knight(0,1), Bishop(0,2), King(0,3), Queen(0,4), Bishop(0,5), Knight(0,6), Rook(0,7)],
+            [Pawn(1,0), Pawn(1,1), Pawn(1,2), Pawn(1,3), Pawn(1,4), Pawn(1,5), Pawn(1,6), Pawn(1,7)],
             [None, None,None,None,None, None,None,None],
+            [None, None,None,None,None, None,None,None],
+            [None, None,None,None,None, None,None,None],
+            [None, None,None,None,None, None,None,None],
+            [Pawn(6,0), Pawn(6,1), Pawn(6,2), Pawn(6,0), Pawn(6,1), Pawn(6,2), Pawn(6,0), Pawn(6,1)],
```
Comment (jerivasuaq) [inline chessboard.py]: Hey pal, you are missing the player #2

**Review by Daniel-Rocha-R — COMMENTED** [review]
lacked of player 2 On code, thus make the changes please

**Review by jerivasuaq — COMMENTED** [review]
Hey @Frank-Garer @MSmartine01 @Kri3gtark I added player 2. Please review it.

**Review by Frank-Garer — APPROVED** [review]

**Pull request closed without merging**